input/list_test.go

```diff
@@ -74,6 +75,10 @@ func makeTestLog(tb testing.TB, dir, fn string, numlines int) string {
 	return fn
 }
 
+func pathToURI(p string) string {
+	return fmt.Sprintf("file://%s", filepath.ToSlash(p))
```

Comment:
nitpicking: maybe this slightly easier to read?
```suggestion
	return "file://" + filepath.ToSlash(p)
```